gatsby-config.js

```diff
@@ -279,6 +281,23 @@ module.exports = {
         nodesPerFeedFile: Infinity,
       },
     },
+    {
+      resolve: 'gatsby-plugin-generate-rss',
+      options: {
+        feeds: [
+          {
+            title: 'Agent Release Notes',
+            path: 'agent-release-notes.xml',
+            regex: '/docs/release-notes/agent-release-notes/',
+          },
+          {
+            title: 'Mobile Release Notes',
```

Comment:
Just to clarify one more time on this requirement:

>  each agent's (including mobile) release notes includes an RSS feed

I'm understanding that to be an RSS feed _per agent_. While the agents are categorized under "Agent release notes", and "Mobile release notes", I'd expect to see one RSS feed per agent. For example "Ruby agent", "PHP agent", "Java agent", etc.

Am I misunderstanding that requirement? Are we just bucketing the agents together into a single RSS feed instead?

Reply:
That's accurate...I think we would like a feed for each agent.

I wanted to push up my work in the current state so we could discuss whether or not we liked the plugin configuration as it is right now. If so, we will want to add a `feed` for _each_ agent in `gatsby-config.js`. (option 🅰️)

An alternative approach would be to handle all of that by the plugin (finding all the agents and making the individual feeds dynamically). This take out all the configuration requirements, but it would be a little less flexible. (option 🅱️)

I'm inclined to keep _some_ of the config (option 🅰️). Thoughts?

Reply:
Just pushed up the configs for all the agents, assuming we're alright with option 🅰️

Reply:
Gotcha! I'm a little slow today so pardon me for missing the intent 😄 

I'd actually be curious to understand the differences between the agents. We can get a distinct set of release notes by type (`subject`) and get their associated versions. If they all conform to the same "interface", having the plugin auto detect new subjects would be pretty nice (🅱️ ). 

That being said, maybe we need the flexibility per agent. Could you walk me through what we might need that could be unique to 1 agent vs another?